### PR TITLE
Display & import images

### PR DESCRIPTION
**Pull request by pierreetiennealary**
display images : doc on experimental-organization.md
import images : doc on the new ws-client.md file

### CODE REVIEW

File: docs/ws-clients.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+The URL of your documentation link should be of the form `'http://[...]/[occurence name]/api-docs/'`. The host name you need to provide on row 17 of `post.py` is this URL but with "api-docs" replaced by "rest" : `'http://[...]/[occurence name]/rest/'`.
+Do not forget the single brackets.
+
+**Row n° 27 & 28** :
+
+Rows 27 and 28 are used for indicating your own login, the one you use when connecting to PHIS via a web browser.
+
+On row 27, replace the second occurence of 'username' by your username between single brackets : it should be an email adress, e.g. `'guestphis@supagro.inra.fr'`.
+
+On row 28, either replace the second occurence of 'password' by your password between single brackets, either replace `hashlib.md5('password').hexdigest()` by an encrypted version of your password : a md5 checksum of your password (also between single brackets), e.g. `'1a79a4d60de6718e8e5b326e338ae533'` (this is the md5 encryption of "example").
```
Comment (Gouga34) [inline docs/ws-clients.md]: The user can add its password in clear and does not need to generate himself the md5 encryption of the password, the "hashlib.md5('password').hexdigest()" will generate the md5 of the password

File: docs/ws-clients.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The first row of the CSV file displays its header :
+
+| path_image_ref     | imageType     | concernedUri     | concernedType     | position    | date     |  sensorUri   |
+| :------------- | :------------- | :------------- | :------------- | :------------- | :------------- | :------------- |
+
+Then, each row of this CSV file is associated to an image imported in PHIS.
+Each image is characterized by :
+- **path_Image_ref** : the name and path of the image (e.g. `image001.png`)
+- **ImageType** : the URI associated with the type of the image (e.g. `http://www.phenome-fppn.fr/vocabulary/2017#HemisphericalImage` or `http://www.phenome-fppn.fr/vocabulary/2017#RGBImage`)
```
Comment (Gouga34) [inline docs/ws-clients.md:50]: The list of images types will be updated and can be found in the vocabulary page of the Phis webapp.

**Review by Gouga34 — CHANGES_REQUESTED** [review]
request some small corrections

File: docs/ws-clients.md (hunk 2 above)
Comment (pierreetiennealary) [reply docs/ws-clients.md:50]: corrected with the commit [1ac5e89](https://github.com/OpenSILEX/phis-docs-community/pull/28/commits/1ac5e89939d583800729e5ba4a4c052fe22b4f11)

**Pull request merged**